lib/rules/no-unnecessary-curly-parens.js

```diff
@@ -0,0 +1,28 @@
+import { builders as b } from 'ember-template-recast';
+
+import Rule from './_base.js';
+import replaceNode from '../helpers/replace-node.js';
+
+export default class NoUnnecessaryCurlyParens extends Rule {
+  visitor() {
+    return {
+      MustacheStatement(node, path) {
+        if (node.path.type === 'SubExpression') {
+          if (this.mode === 'fix') {
+            replaceNode(
+              node,
+              path.parentNode,
+              path.parentKey,
+              b.mustache(node.path.path, node.path.params, node.path.hash)
+            );
+          }
+          this.log({
+            node,
+            isFixable: true,
+            message: 'Unnecessary parentheses enclosing statement',
+          });
```

Comment:
Can you move this into an `else`? Not sure if it's strictly required but all our other rules follow `if (this.mode === 'fix') { } else { this.log() }`.